docs/pact-reference.md

```diff
@@ -513,7 +516,7 @@ Examples of valid keyset JSON productions:
 
 ### Keyset Predicates {#keyset-predicates}
 
-A keyset predicate references a function by name which will compare the public keys in the keyset
+A keyset predicate references a function by optionally-qualified name which will compare the public keys in the keyset
```

Comment:
this reads a bit funny. maybe "A keyset predicate references a function by its (optionally qualified) name"